lib/active_resource/connection.rb

```diff
@@ -18,6 +18,13 @@ class Connection
       delete: "Accept",
       head: "Accept"
     }
+    HTTP_METHODS = {  get: Net::HTTP::Get,
```

Comment:
```suggestion
    HTTP_METHODS = {
      get: Net::HTTP::Get,
```

and fix the one above as well in a different commit